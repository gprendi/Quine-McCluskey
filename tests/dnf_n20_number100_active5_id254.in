d 20
00001010000000000010
00000010100000000000
00000010100000000010
00001001000000000010
00000000000000000000
00001011000000000010
00000001100000000010
00001000000000000010
00000010000000000010
00000011100000000010
00000001000000000000
00000001000000000010
00001011100000000000
00001001100000000010
00001000000000000000
00001010100000000010
00001001100000000000
00001000100000000000
00000001100000000000
00001011100000000010
00000011000000000010
00001001000000000000
00001011000000000000
00000000100000000010
00000011000000000000
00000000000000000010
00001000100000000010
00001010000000000000
00000010000000000000
00000011100000000000
00000000100000000000
00001010100000000000
