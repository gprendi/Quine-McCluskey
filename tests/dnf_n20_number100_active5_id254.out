p dnf 20 0
-1 -2 -3 -4 -6 -10 -11 -12 -13 -14 -15 -16 -17 -18 -20 0
